participants/article-memo.md

```diff
@@ -0,0 +1,61 @@
+# <article-memo>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+*Our project name is article-memo. Our team aims to provide service for filecoin miner and also we're developing applications for distributed storage*
+
+## (1) Brief Project Description
+
+*This project is to host public academic aritcles for everyone, and also provide a platform such that people share their articles to each other.*
+
+## (2) Link to Project UI
+
+*This project is hosted on https://github.com/yydfjt/article-memo.git*
+
+## (3) What does your application/UI do?
+
+*Users can get and upload articles from our application*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*https://arxiv.org/*
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*Our data consists two parts: public articles on website such as https://eprint.iacr.org/;  and also upload by people.*
```

Comment:
If you are submitting IACR data please add it to the curated dataset so you can receive full credit for this data source as well.